src/app/shared/components/select/select.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container pb-2 pt-2">
-    <label *ngIf="label">{{ label }}</label>
+    <label class="flex" *ngIf="label">{{ label }}<sofie-form-required-mark *ngIf="isRequired"></sofie-form-required-mark></label>
```

Comment:
Why is this marked as flex?

Reply:
It was because the `*` was moving to the new row, but after we changed the `sofie-required-mark` structure it is no longer necessary